app/views/auctions/_bids_and_form.html.erb

```diff
@@ -15,3 +15,7 @@
     </span>
   </div>
 </div>
+
+<div class="issue-bids-info-item">
+  <%= render partial: auction.bid_button_partial, locals: { auction: auction.auction } %>
```

Comment:
This `auction.auction` looks a bit weird. Is this to get to an less-wrapped version of the auction object? Can we give that a more descriptive name?